android/app/src/main/kotlin/com/leastauthority/dart_wormhole_gui/MainActivity.kt

```diff
@@ -28,6 +31,31 @@ class MainActivity : FlutterActivity() {
     private val READ_FAILURE = "5"
     private val CLOSE_FAILURE = "6"
 
+    override fun onCreate(savedInstanceState: Bundle?) {
+        when {
+            intent?.action == Intent.ACTION_SEND -> {
```

Comment:
This is probably related with another PR for enabling sharing with app.

Reply:
@donpui Sorry i pushed AndroidManifest.xml and MainActivity.kt by accident
It is temp branch, I did not know we are going to review it.